presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1142,7 +1145,22 @@ public SystemSessionProperties(
                         EXCEEDED_MEMORY_LIMIT_HEAP_DUMP_FILE_DIRECTORY,
                         "Directory to which heap snapshot will be dumped, if heap_dump_on_exceeded_memory_limit_enabled",
                         System.getProperty("java.io.tmpdir"),   // This is intended to be used for debugging purposes only and thus we does not need an associated config property
-                        true));
+                        true),
+                booleanProperty(
+                        SMART_SAMPLING_ENABLED,
+                        "Smart sampling of tables enabled for adhoc queries",
```

Comment:
Please give a description so a common Presto user can read and understand what this is about.

Reply:
Done